### ISSUE TITLE
Great code, missing documentation

### ISSUE THREAD

**Issue by kwboone** [DBCG/cql_execution_service#4]
The code is clear and understandable, however, use of it would be greatly assisted by JavaDoc and some user oriented documentation.  I've never really worked on an open source project other than posting bugs, but I might be interested in helping out with the documentation challenges on this code base.

**Comment by c-schuler** [comment]
I agree that formal documentation would improve the user's experience and understanding with this project. As an open source project we encourage and appreciate contributions from the community to enhance and harden the code base.

**Comment by brynrhodes** [comment]
Hi kwboone! Yep, the documentation is a soft spot for sure. We'd love any help you'd be willing to provide, let me know if you have any questions on how to contribute.

**Comment by kwboone** [comment]
Lots.  I’ll skype you next week to set up a time to discuss
